### DISCUSSION TITLE
How should we go about sources cleanup?

### DISCUSSION THREAD

**Discussion by lppedd** [eclipse-tycho/tycho#1690] [Q&A] [answered]
What are the current rules for sources? E.g. are we allowed to use type inference now?
When we touch a piece of code, should we reformat it first, or just do the change?

**Comment by laeubi** [comment, marked as answer]
One could use whatever is allowed by the language and makes the code readable.

If changes are required/desired that are not in the scope of the change it usually is better to create this in an own commit.

**Reply by lppedd** [reply]
Perfect. Thanks!
